pkg/constants/constants.go

```diff
@@ -36,9 +36,9 @@ const (
 	IngressControllerPrefix       = "kubesphere-router-"
 
 	WorkspaceLabelKey              = "kubesphere.io/workspace"
-	DisplayNameAnnotationKey       = "displayName"
-	DescriptionAnnotationKey       = "desc"
-	CreatorAnnotationKey           = "creator"
+	DisplayNameAnnotationKey       = "kubesphere.io/display-name"
```

Comment:
since we've already used kubesphere.io/alias-name,   https://tower.im/teams/637817/repository_documents/8362/, better to align with current usage